### PR TITLE
Allows a team to join a tournament

### PR DESCRIPTION
**Pull request by chf3kl9**
requires Tournamanager/api#142

### CODE REVIEW

**Comment by MindHawk** [comment]
Tournaments are now properly gotten

removeTeamFromTournament does not exist on the backend, but is called

File: src/components/team/TeamTournamentsComponent.jsx [hunk 1]
```diff
@@ -1,26 +1,73 @@
...
+                        <li key={item.id} className="list-group-item">
+                            {item.name}
+                            <button
+                                className="btn-danger"
+                                onClick={() => this.leaveTournament(item.id)}
+                            >
+                                x
+                            </button>
+                        </li>
+                    ))}
```
Comment (MichielLangen) [inline src/components/team/TeamTournamentsComponent.jsx]: This renders double the tournaments 🙅‍♂ 🚫

Comment (MindHawk) [reply src/components/team/TeamTournamentsComponent.jsx]: Fixed by not calling `getAllTournamentsThisTeamIn()`
This was a fix when tournaments were not contained in the team class

**Comment by MindHawk** [comment]
Works.

**Pull request merged**